sdk/communication/AzureCommunicationChat/Tests/ChatClientTests.swift

```diff
@@ -33,19 +33,19 @@ class ChatClientTests: XCTestCase {
     /// ChatClient initialized in setup.
     private var chatClient: ChatClient!
     /// Test mode.
-    private var mode = ProcessInfo.processInfo.environment["TEST_MODE"] ?? "playback"
+    private var mode = getEnvironmentVariable(withKey: "TEST_MODE", default: "playback")
```

Comment:
We could define the environment variable names in one place like an enum or struct instead of hardcoding them multiple times as I did, since only certain ones are valid anyways

Reply:
True. I made this change mainly because I wanted the environment variable names to remain in the scheme settings, which meant the previous way would return empty string instead of nil. As we look for ways to make this more generic, we would ideally be able to entirely do away with environment variables.